package.json

```diff
@@ -1,36 +1,28 @@
 {
-  "version": "1.9.2",
   "devDependencies": {
-    "@fitbit/sdk": "^4.2.1",
-    "@fitbit/sdk-cli": "^1.7.3"
+    "@fitbit/sdk": "~5.0.0"
   },
   "fitbit": {
-    "appUUID": "19a118e0-3070-4f00-b76b-345d02853c70",
+    "appUUID": "c7514968-715d-48d3-927b-4b23c54e5f71",
     "appType": "clockface",
     "appDisplayName": "Roni",
     "iconFile": "resources/icon.png",
     "wipeColor": "",
     "requestedPermissions": [
       "access_activity",
       "access_heart_rate",
-      "access_internet"
+      "access_internet",
+	  "run_background"
```

Comment:
Just curiour, what did adding "run_background" do?  Do we need it for something?